students/hangu/westagram/urls.py

```diff
@@ -0,0 +1,6 @@
+
+from django.urls import path
+
+urlpatterns = [
+
+]
```

Comment:
```suggestion
urlpatterns = [
]
```

파이썬은 모든 라인을 scan해서 실행합니다. 그렇기 때문에 불필요하게 공백을 남겨놓지 않도록 주의해주세요.